packages/eslint-plugin/__tests__/stylex-margin-padding.js

```diff
@@ -35,6 +35,26 @@ eslintTester.run('stylex-sort-keys', rule.default, {
       })
     `,
     },
+    {
+      code: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        main: {
+          margin: 10,
+        },
+      })
+    `,
+    },
+    {
+      code: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        main: {
+          padding: 'calc(0.5 * 100px) !important',
```

Comment:
Please can we remove `!important` from these fixtures so we don't need to modify them when `!important` support is removed.

Reply:
synced with @nmn and we decided to leave this in as a "might as well have", but if we're planning to remove !important support in the future than I agree it may make sense to remove from the tests

Reply:
@mellyeliu Yes, remove `!important` from the tests. We should just account for splitting up values even when `!important` is present. This is mostly a defensive thing to account for stuff in our codebase.